src/components/home/roadmapsolving.tsx

```diff
@@ -2,78 +2,85 @@ import React, { FC } from 'react';
-      <Box
-        sx={{
-          position: 'absolute',
-          top: 0,
-          left: '50%',
-          transform: 'translateX(-50%)',
-          width: '4px',
-          height: '1050px',
-          background: 'linear-gradient(to bottom, transparent, #0066FF 40px, #0066FF)',
-          zIndex: 1,
-        }}
-      />
-      {/* Vertical Line with Arrow */}
-      <Box
-        sx={{
-          position: 'absolute',
-          left: '50%',
-          bottom: '-20px',
-          transform: 'translate(-50%, 0)',
-          width: 0,
-          height: 0,
-          borderLeft: '10px solid transparent',
-          borderRight: '10px solid transparent',
-          borderTop: '20px solid #0066FF',
-          borderBottomLeftRadius: '-10px',
-          borderBottomRightRadius: '-10px',
-          zIndex: 1,
-        }}
-      />
+      <Box sx={{ position: 'relative', width: '100%', overflow: 'visible' }}>
+        {/* Central Vertical Line - Hidden on mobile */}
+        {!isMobile && (
+          <>
+            <Box
+              sx={{
+                position: 'absolute',
+                top: 0,
+                left: '50%',
+                transform: 'translateX(-50%)',
+                width: '4px',
+                height: '1230px',
+                background: 'linear-gradient(to bottom, transparent, #0066FF 40px, #0066FF)',
+                zIndex: 1,
+              }}
+            />
+            <Box
+              sx={{
+                position: 'absolute',
+                left: '50%',
+                bottom: '-20px',
+                transform: 'translate(-50%, 0)',
+                width: 0,
+                height: 0,
+                borderLeft: '10px solid transparent',
+                borderRight: '10px solid transparent',
+                borderTop: '20px solid #0066FF',
+                borderBottomLeftRadius: '-10px',
+                borderBottomRightRadius: '-10px',
+                zIndex: 1,
+              }}
+            />
+          </>
+        )}
 
-      {data.map((item, index) => (
-        <Box
-          key={item.id}
-          sx={{
-            display: 'flex',
-            flexDirection: index % 2 === 0 ? 'row-reverse' : 'row',
-            alignItems: 'center',
-            mb: 10,
-            position: 'relative',
-            width: '100%',
-          }}
-        >
-          {/* Circle on the vertical line */}
-                <Box
+        {data.map((item, index) => (
+          <Box
+            key={item.id}
+            sx={{
+              display: 'flex',
+              flexDirection: isMobile ? 'column' : (index % 2 === 0 ? 'row-reverse' : 'row'),
+              alignItems: isMobile ? 'flex-start' : 'center',
+              mb: 10,
+              position: 'relative',
+              width: '100%',
+            }}
+          >
+            {/* Circle - Hidden on mobile */}
+            {!isMobile && (
+              <Box
```

Comment:
```suggestion
        {data.map((item, index) => (
          <Box
                  key={item.id}
                  sx={{
                    display: 'flex',
                    flexDirection: isMobile ? 'column' : (index % 2 === 0 ? 'row-reverse' : 'row'),
                    alignItems: isMobile ? 'flex-start' : 'center',
                    mb: 10,
                    position: 'relative',
                    width: '100%',
                  }}
                >
                  {/* Circle - Hidden on mobile */}
                  {!isMobile && (
              <Box
```
Are we not using a formatter? we should have formatter in make file. In tsx, maybe `npm run format` is the comand to format.

Reply:
We also need to have make file, and a single `make ci` comand should format, test and execute all other nacessary comands. Therefore, before we push, we can just simply execute `make ci` and push.

Reply:
Added make file and did format the codebase